test/render_html.py

```diff
@@ -42,6 +42,9 @@ def render_html(themes: list, languages=languages, outdir=outdir):
             Defaults to outdir.
     """
 
+    if not themes:
```

Comment:
Decided to move the logic into the function so that if `render_html()` is called without arguments, it builds the syntax-highlighting sample HTML pages for all of the themes.